cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -235,29 +238,27 @@ def test_course_listing_performance(self):
         self.assertGreaterEqual(iteration_over_courses_time_2.elapsed, iteration_over_groups_time_2.elapsed)
 
         # Now count the db queries
-        with check_mongo_calls(USER_COURSES_COUNT):
+        with check_mongo_calls(150):
             _accessible_courses_list_from_groups(self.request)
 
+        with check_mongo_calls(505):
+            _accessible_courses_list(self.request)
         # Calls:
         #    1) query old mongo
         #    2) get_more on old mongo
         #    3) query split (but no courses so no fetching of data)
-        with check_mongo_calls(3):
```

Comment:
Not for this PR, but thinking about this more, we should probably deprecate and then remove these methods that make way too many queries.